assets/wallet/mytonwallet.json

```diff
@@ -15,6 +15,14 @@
 					"tags": ["pool"],
 					"submittedBy": "rAndom1ze",
 					"submissionTimestamp": "2025-03-11T00:00:01Z"
+			},
+			{
+				"address": "EQDUkQbpTVIgt7v66-JTFR-3-eXRFz_4V66F-Ufn6vOg0D5s",
```

Comment:
@ohld may be it should come to the trading bot category?

Reply:
To be honest, I'm not sure. I think we can add a tag '"referral-address"' here, but I don't think it is necessary.